### PR TITLE
Added custom fields "placement" attribute

### PR DESCRIPTION
**Pull request by jan-vince**
This PR adds attribute **placement=""** to allow custom fields placement in the primary tab container when content of custom field is settings related.

````
{variable name="tagline" label="Tagline" tab="Header" type="text" placement="primary"}{/variable}
````

README.md file is updated with description and example.

I have removed few unnecessary white spaces in edited files.

### CODE REVIEW

File: controllers/Index.php [hunk 1]
```diff
@@ -434,8 +434,20 @@ protected function addPageSyntaxFields($formWidget, $page)
                 unset($fieldConfig['fields']);
             }
 
-            $fieldConfig['cssClass'] = 'secondary-tab ' . array_get($fieldConfig, 'cssClass', '');
-            $formWidget->secondaryTabs['fields']['viewBag[' . $fieldCode . ']'] = $fieldConfig;
+			$placement = (!empty($fieldConfig['placement']) ? $fieldConfig['placement'] : NULL);
```
Comment (LukeTowers) [inline controllers/Index.php]: Spaces, not tabs please. I prefer tabs myself too, but everything in October is spaces, so please follow that convention.

File: controllers/Index.php [hunk 2]
```diff
@@ -434,8 +434,20 @@ protected function addPageSyntaxFields($formWidget, $page)
...
+
+			switch($placement) {
+
```
Comment (LukeTowers) [inline controllers/Index.php]: Formatting consensus for switch statements is as follows:
```php
switch ($value) {
    case 'condition1':
         // do stuff
         break;
    case 'condition2':
         // do stuff
         break;
    default:
        // do stuff
        break;
}
```

Comment (jan-vince) [inline controllers/Index.php]: Ok, I have fixed this.

_And this is my first PR so please let me know if simply add new commit is enough. Thanks :)_

**Comment by seanthepottingshed** [comment]
@jan-vince, @LukeTowers thank you very much for your hard work

**Comment by pavsid** [comment]
@jan-vince thanks for this. Is there any way to add a field to the existing 'Settings' tab? If I specify `tab="Settings"` then a second 'Settings' tab appears in the primary tabs.

**Comment by LukeTowers** [comment]
@pavsid use `cms::lang.editor.settings` instead of `Settings`

**Pull request merged**